srcpkgs/nheko/template

```diff
@@ -1,26 +1,33 @@
 # Template file for 'nheko'
 pkgname=nheko
-version=0.11.3
+version=0.12.0
 revision=1
 build_style=cmake
-hostmakedepends="qt5-host-tools qt5-qmake pkg-config qt5-declarative asciidoc"
-makedepends="qt5-multimedia-devel qt5-svg-devel qt5-tools-devel fmt-devel
+hostmakedepends="qt6-base qt6-tools qt6-declarative-tools pkg-config asciidoc"
```

Comment:
You most likely only need `qt6-declarative-host-tools` and  can drop the workaround in `pre_build` that way